examples/basic/variables.tf

```diff
@@ -1,33 +1,35 @@
-########################################################################################################################
-# Input variables
-########################################################################################################################
+####################
+# Cluster variables
+####################
+variable "cluster_id" {
+  type        = string
+  description = "The name or ID of the cluster to be used"
+  default     = "my-cluster-name"
+}
 
 variable "ibmcloud_api_key" {
   type        = string
-  description = "The IBM Cloud API Key"
+  description = "IBM Cloud API key"
   sensitive   = true
 }
 
-variable "region" {
+variable "ibmcloud_region" {
   type        = string
-  description = "Region to provision all resources created by this example"
+  description = "IBM Cloud region where all resources will be deployed"
   default     = "us-south"
 }
 
-variable "prefix" {
-  type        = string
-  description = "Prefix to append to all resources created by this example"
-  default     = "basic"
-}
-
-variable "resource_group" {
```

Comment:
I suggest to add this back in and have your example support creating a resource group, or using an existing one. Just like it has here -> https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/fac073820ea9db73fd02fc4d738d6bdcb01a27b3/examples/basic/main.tf#L5-L11